include/elements/2d/quadrilateral_lme_element.tcc

```diff
@@ -0,0 +1,382 @@
+//! Assign nodal connectivity property for LME elements
+template <unsigned Tdim>
+void mpm::QuadrilateralLMEElement<Tdim>::initialise_lme_connectivity_properties(
+    double beta, double radius, const Eigen::MatrixXd& nodal_coordinates) {
+  assert(nodal_coordinates.rows() == nodal_properties.size());
+
+  this->nconnectivity_ = nodal_coordinates.rows();
+  this->nodal_coordinates_ = nodal_coordinates;
+  this->beta_ = beta;
+  this->support_radius_ = radius;
+}
```

Comment:
How about if we order in the first n elements of nodal_coordinates the nodes which lays inside of the circle? Then, for each particle at the time we just have to evaluate the shape function for the first n nodes. We need an integer n for each particle, this is possible?

Reply:
Agree